pandas/tests/series/methods/test_tolist.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import pandas.util._test_decorators as td
+
+from pandas import (
+    Interval,
+    Period,
+    Series,
+    Timedelta,
+    Timestamp,
+)
+
+
+@pytest.mark.parametrize(
+    "values, dtype, expected_dtype",
+    (
+        ([1], "int64", int),
+        ([1], "Int64", int),
+        ([1.0], "float64", float),
+        ([1.0], "Float64", float),
+        (["abc"], "object", str),
+        (["abc"], "string", str),
+        ([Interval(1, 3)], "interval", Interval),
+        ([Period("2000-01-01", "D")], "period[D]", Period),
+        ([Timedelta(days=1)], "timedelta64[ns]", Timedelta),
+        ([Timestamp("2000-01-01")], "datetime64[ns]", Timestamp),
+        pytest.param(
+            [1],
+            "int64[pyarrow]",
+            int,
+            marks=td.skip_if_no("pyarrow", min_version="1.0.0"),
```

Comment:
I think min version is 7 or something similar, please adjust accordingly

Reply:
updated to version 6 which is the min version tested, although this test should pass with any version. It looks like there are a handful of other tests that have less than 6.0.

Reply:
I think you can get rid of the min version then

Reply:
ah, thanks. updated